pkg/apis/apps/types.go

```diff
@@ -44,6 +44,21 @@ type StatefulSet struct {
 	Status StatefulSetStatus
 }
 
+// PodCreationPolicyType defines the policy for creating pods under a stateful set.
+type PodCreationPolicyType string
+
+const (
+	// OrderedReadyPodCreation will create pods in strictly increasing order on
+	// scale up and strictly decreasing order on scale down, progressing only when
+	// the previous pod is ready or terminated. At most one pod will be changed
+	// at any time.
+	OrderedReadyPodCreation PodCreationPolicyType = "OrderedReady"
+	// ParallelPodCreation will create and delete pods as soon as the stateful set
```

Comment:
I veto crud just because it has poor context.  What is the concept that ties all of those together without acryonymizing it?